docs/source/en/main_classes/quantization.md

```diff
@@ -128,12 +128,21 @@ For 4-bit model, you can use the exllama kernels in order to a faster inference
 
 ```py
 import torch
-gptq_config = GPTQConfig(bits=4, disable_exllama=False)
+gptq_config = GPTQConfig(bits=4, disable_exllama=False, disable_exllamav2=True)
+model = AutoModelForCausalLM.from_pretrained("{your_username}/opt-125m-gptq", device_map="auto", quantization_config = gptq_config)
+```
+With the release of the exllamav2 kernel, you can get faster inference speed compared to the exllama kernels. You just need to 
+pass `disable_exllamav2` in [`GPTQConfig`]:
```

Comment:
it doesn't make a lot of sense to have a `disable_exllamav2` instead of `enable_` or `use_` which we have for most of the additional features like flash attention or `use_cache`. No?

Reply:
It is named it this way because the goal was to enable it by default (`False` value). Since the v2 came out, I decided to disable the v1 and enable v2 by default. cc @fxmarty as you did the v1 integration.

Reply:
Yes I agree the name is not very scalable. It could make sense to rename them.

Reply:
Renamed !